### PR TITLE
rewrites errors

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/3091.

### CODE REVIEW

**Review by kopiczko — APPROVED** [review]

File: go/errors.md [hunk 1]
```diff
@@ -96,7 +101,10 @@ func (err AlreadyRegisteredError) Error() string {
 	return "duplicate metrics collector registration attempted"
 }
 ```
+
+
 Which we can align with our pattern easily like this:
+
 ```go
 var alreadyRegisteredError = microerror.New("already registered")
```
Comment (indyfree) [inline go/errors.md]: Can we use the new pattern here too?

File: go/errors.md [hunk 2]
```diff
@@ -96,7 +101,10 @@ func (err AlreadyRegisteredError) Error() string {
 	return "duplicate metrics collector registration attempted"
 }
 ```
+
+
```
[the remark this answers is not in the document]
Comment (xh3b4sd) [reply go/errors.md]: No, fixed.

File: go/errors.md (hunk 1 above)
Comment (xh3b4sd) [reply go/errors.md]: Yes, done.

**Comment by xh3b4sd** [comment]
PTAL

**Review by indyfree — APPROVED** [review]
Cool! Thanks for your efforts to push this through.

**Comment by xh3b4sd** [comment]
Thanks for your thorough reviews. <3

**Pull request merged**